src/wallet/wallet2.cpp

```diff
@@ -6314,7 +6314,7 @@ bool wallet2::sign_tx(unsigned_tx_set &exported_txs, std::vector<wallet2::pendin
     // normally, the tx keys are saved in commit_tx, when the tx is actually sent to the daemon.
     // we can't do that here since the tx will be sent from the compromised wallet, which we don't want
     // to see that info, so we save it here
-    if (store_tx_info() && ptx.tx_key != crypto::null_skey)
+    if (store_tx_info() && tx_key != crypto::null_skey)
```

Comment:
I believe we don't need the `tx_key != crypto::null_skey` check either.

Reply:
I think you're right, but it got added to be consistent with the other places, it does no harm I think.